### DISCUSSION TITLE
Jellyfin Problem collection

### DISCUSSION THREAD

**Discussion by Fabio89160** [jellyfin/jellyfin#9721] [General]
bonjour j'ai un soucis avec les collections, quand un film apparait avec "&" dans le titre jellyfin crée plusieurs collections avec "&amp" dans le titre comment puis-je regler ce problème SVP

**Comment by Weels92** [comment]
Salut !

En admettant que tes fichiers d'une collection fassent partit d'un dossier.
Effacer tous les fichiers (autre que les vidéos) dans le dossier, renommer proprement les fichiers vidéos et relancer un scan.
